### PR TITLE
Activates readonly option

### PR DESCRIPTION
**Pull request by 6temes**
The :readonly option was not working, so I made a couple of changes in order to activate it.

### CODE REVIEW

File: lib/ratyrate/helpers.rb [hunk 1]
```diff
@@ -25,15 +25,17 @@ def rating_for(rateable_obj, dimension=nil, options={})
     targetType   = options[:targetType]   || 'hint'
     targetFormat = options[:targetFormat] || '{score}'
     targetScore  = options[:targetScore]  || ''
-    readOnly     = options.delete(:readonly){ false }
+    readonly     = options[:readonly]     || false
```
Comment (wazery) [inline lib/ratyrate/helpers.rb]: Can you please change this to `readOnly` instead of `readonly?

File: lib/ratyrate/helpers.rb [hunk 2]
```diff
@@ -25,15 +25,17 @@ def rating_for(rateable_obj, dimension=nil, options={})
...
 
     disable_after_rate = options[:disable_after_rate] && true
     disable_after_rate = true if disable_after_rate == nil
 
-    if disable_after_rate
-      readonly = !(current_user && rateable_obj.can_rate?(current_user, dimension))
-    else
-      readonly = !current_user || false
+    if !readonly
```
Comment (wazery) [inline lib/ratyrate/helpers.rb]: Can this be changed to `unless readOnly`?

File: lib/ratyrate/helpers.rb [hunk 3]
```diff
@@ -105,10 +107,10 @@ def rating_for_user(rateable_obj, rating_user, dimension = nil, options = {})
     targetType   = options[:targetType]   || 'hint'
     targetFormat = options[:targetFormat] || '{score}'
     targetScore  = options[:targetScore]  || ''
+    readonly     = options[:readonly]    || false
```
Comment (wazery) [inline lib/ratyrate/helpers.rb]: Can you please maintain the alignment of `||` to the above line.

**Comment by wazery** [comment]
@6temes Thanks for your PR, I left some minor comments, please check them, and I will merge ASAP :wink:

**Comment by 6temes** [comment]
Done! 

Thank you! :)

**Comment by wazery** [comment]
Thanks @6temes :)

**Comment by 6temes** [comment]
That was actually my first pull request merged in a OS repo! :smile: :tada: :tada:

**Comment by wazery** [comment]
Then welcome to the OS world! :smiley:

**Pull request merged**